new_arrivals_chi/app/main.py

```diff
@@ -48,7 +49,12 @@ def home():
     """
 
     language = request.args.get("lang", "en")
-    return render_template("home.html", language=language)
+    translations = load_translations()
```

Comment:
Instead of loading them in every function, I'd load them on application load. `translations` would be a variable accessible in every function. It would in theory save you that I/O time.